ros/angel_system_nodes/angel_system_nodes/task_monitoring/global_step_predictor.py

```diff
@@ -400,6 +406,9 @@ def det_callback(self, activity_msg: ActivityDetection):
         conf_array = np.array(activity_msg.conf_vec)
         conf_array = np.expand_dims(conf_array, 0)
 
+        if self._only_manual == True:
+            return
+        
```

Comment:
I believe that this the `only_manual` controls here were also deprecated in lieu of my SystemCommand updates.